### PR TITLE
Portfolio: add a contact form and finish the mobile version

### PR DESCRIPTION
**Pull request by Nelofarzabi**
In this section : 
-Create forms with HTML5 validations.
-Understand the importance of UX.
-Understand how to parse a Figma design to create a UI.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -297,22 +297,48 @@ <h4 class="connect">LETS CONNECT</h4>
...
+        action="https://formspree.io/f/mlevdeoa"
+        method="POST">
+
+          <input type="text" name="user-name" placeholder="Name" class="form-input-name" maxlength="30" required>
+          <input type="email" name="email" placeholder="Email" class="form-input-email" required> 
+          <textarea name="message"  class="text-message" placeholder="Write your message here..." maxlength="500" required></textarea>
+          <div class="submit-btn-div">
+            <button class="Get-In-Touch-btn" type="submit">
+              Get in touch
+            </button>
```
Comment (AlphaNtihinduka) [inline index.html:338]: - [ ] I suggest that we stick to the Figma design as much as possible. I suggest that you ensure that the color and background color of the button is changed in accordance the Figma design provided both in different button state.
Figma design 😎 
image

**Review by AlphaNtihinduka — CHANGES_REQUESTED** [review]
## Hello Team🙋‍♂️
Good job working on your contact form. You have done the design to the best expectation and incredibly well. 👏  

You still need to implement few on the project but you are almost there!!! :watch: 

TO HIGHLIGHT:
- Styling matches the figma design ✔️ 
- Used POST method value correctly ✔️ 
- Used form id ✔️ 
- Well-structured files :heavy_check_mark: 

# STATUS: Required Changes :recycle: 

close

# Required Changes :recycle: 
_- Kindly refer to the inline review comment ☺️._

## Optional suggestions
- [ ] [OPTIONAL]  Consider the use of [`<fieldset>`](https://developer.mozilla.org/en-US/docs/Learn/Forms/How_to_structure_a_web_form#the_fieldset_and_legend_elements)
If you notice for form [accessibility](https://developer.mozilla.org/en-US/docs/Learn/Accessibility) reasons wrapping the inputs in the `<fieldset>` is more semantic. Kindly make the form more accessible to screen readers such us [Jaws](https://www.freedomscientific.com/products/software/jaws/) and [NVDA](https://www.nvaccess.org) by using `<fieldset>`.



_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or slack channel if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAPPY CODING AND GOOD LUCK :crossed_fingers: :four_leaf_clover:

File: style.css [hunk 2]
```diff
@@ -490,6 +490,21 @@ hr {
   cursor: pointer;
 }
 
+.Get-In-Touch-btn:hover {
+  cursor: pointer;
+  background-color: white;
+  box-shadow: 0 0 16px rgba(64, 83, 252, 0.24);
+  transform: translateY(4px);
+  color: #6070ff;
+}
```
Comment (Tufahel) [inline style.css]: - This is not the right implementation for this hover button. `background-color` should be changed to `#6070ff` color but you have used `white` and `color` should be `white` but you have used `#6070ff`.
image
- [ ] Kindly fix this issue to match the given design.

**Review by Tufahel — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi @Nelofarzabi & @AlyAtalla :wave:  

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 🌟

Alt Text

✔️ Descriptive pull request.
✔️ Linter checks are passing.
✔️ Follow the correct GitHub flow.
✔️ Commit messages are meaningful.
✔️ UI is decent.

## Required Changes ♻️

***Please kindly check the comments/review below.***

## Optional suggestions 🤷‍♂️

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css
Comment (KanzaTahreem) [inline style.css:1]: - [OPTIONAL] Please make sure that the website is responsive, following these breakpoints as mentioned in the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md) 👇 For this, you can use responsive values in your CSS rules, and avoid using a fixed width, `max-width: 375px;`, under `body` selector ✨⛏ This approach will also be helpful when you work on the desktop version of the portfolio. 💪
  - Minimum mobile screen size should be `375px`
  - Minimum desktop screen size should be `768px`

**Review by KanzaTahreem — APPROVED** [review]
# Approved 🥳

Hi @Nelofarzabi 🙋‍♀️

Great work implementing the suggested changes 💪 **_Your project is COMPLETE! There is nothing else to say other than... it's time to merge it! 🚀 Congratulations!_**



## Highlights 🎯
There are some excellent points that you have made perfectly. Keep up the great work! 👏

-  ✅ Styling is looking great, perfect!
-  ✅ Great work using the right GitHub Flow 
-  ✅ Your README looks good, well done!
-  ✅ Linters are working properly, great job! 

## Optional suggestions ❕
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

N/A

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**